### ISSUE TITLE
Discover pane in opensearch provides a hint for longer logs

### ISSUE THREAD

**Issue by gautam-nutalapati** [opensearch-project/OpenSearch-Dashboards#2059]
**Is your feature request related to a problem? Please describe.**

When I search for logs in discover pane, for longer logs, I have no clue if thats the end of log or if I need to expand the log to see full log content.

**Describe the solution you'd like**

A hint that the log is expandable. Show `<expand>` OR `...` at end of unexpanded log line.

**Describe alternatives you've considered**

A dropbox`expand` that can take in document fields for whom expanded contents will be shown in default discover view.

**Additional context**

Attached are screenshot of how I expand a log today.
<img width="1984" alt="Screen Shot 2022-08-03 at 12 11 18 PM" src="https://user-images.githubusercontent.com/81705313/182670501-bf8fb78b-3c1e-4863-bdb5-8ca102636141.png">
<img width="1954" alt="Screen Shot 2022-08-03 at 12 12 13 PM" src="https://user-images.githubusercontent.com/81705313/182670504-481f3cb4-be60-4829-b54d-bb8406f64ca8.png">

**Comment by ananzh** [comment]
Yeah, we just don’t have any formatter in discover to format long log or any contents.  I think this is an enhancement feature/improve for cx. @gautam-nutalapati , could you help us to understand what is ur expectation or what do you think is more convenient for your usage? 

@KrooshalUX also want your thoughts on this.

**Comment by KrooshalUX** [comment]
@shanilpa - getting this on your radar in regards to the upcoming Data Explorer.

**Comment by AMoo-Miki** [comment]
The only sane way of doing this is to forgo the height-based clipping in favor of line clamping.
